examples/speed_estimation/ultralytics_example.py

```diff
@@ -70,7 +70,7 @@ def parse_arguments() -> argparse.Namespace:
     args = parse_arguments()
 
     video_info = sv.VideoInfo.from_video_path(video_path=args.source_video_path)
-    model = YOLO("yolov8x.pt")
+    model = YOLO("yolo11x.pt")
```

Comment:
let's not do it. I want to make sure people will get the same outputs as I did during speed estimation demo